src/ws/update-motd

```diff
@@ -18,7 +12,32 @@ ip=${3:-$(ip -o route get 255.0 2>/dev/null | sed -e 's/.*src \([^ ]*\) .*/\1/')
 # protocol from cmdline, then https
 protocol=${4:-https}
 
-hostname_url="${protocol}://${hostname}:${port}/"
-ip_url="${ip:+ or ${protocol}://${ip}:${port}/}"
+port_only="\[::\]*"
+# If the address starts with a slash ("/"), it is read as file system socket
+socket_path="\/*"
+empty_string=""
 
-printf 'Web console: %s%s\n\n' "${hostname_url}" "${ip_url}"  > /run/cockpit/active.motd
+printf 'Web console: \n' > /run/cockpit/active.motd
+# port number from cmdline, then systemctl file
+for line in ${1:-$(systemctl show --property Listen cockpit.socket)}; do
```

Comment:
Or there is some mess with spaces and tabs here - therefore it appears wrongly aligned. I will fix together with the review. 
Do we have some convention regarding using tabs for shell scripts?

Reply:
Better avoid tabs completely.